core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -25,9 +25,9 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
 
   <j:invokeStatic var="ds" className="jenkins.model.DownloadSettings" method="get"/>
-  <form method="post" action="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}">
-    ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  <f:link href="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}" post="true">
     <f:submit value="${%Check now}" />
```

Comment:
without this #@%! YUI stuff I'd just have used onclick="this.form.action='checkUpdates'". If you know which css class to use to get button look'n feel on a link, I'd be happy to remove this submit button.